ansible/roles/flask_app_config/templates/config.sh

```diff
@@ -89,3 +89,37 @@ REMS_LOGOUT_MESSAGE="{{ session_check_rems_logout_message }}"
 
 # log file fron "session expiry check" cron job
 CRON_SESSION_EXPIRE_LOG="{{ session_check_cron_session_expire_log }}"
+
+
+### Request restriction configurations
+
+# Sliding windows for request limits
+SHORT_TIMEFRAME="{{ flask_app.short_timeframe }}" 
+LONG_TIMEFRAME="{{ flask_app.long_timeframe }}"
```

Comment:
Should these be without double quotes (") if we assume it is legitime practice for integers in this file?